aiida/orm/nodes/node.py

```diff
@@ -969,6 +767,15 @@ def get_description(self) -> str:
         'update_comment': 'update',
     }
 
+    _deprecated_links_methods = {
```

Comment:
You forgot to actually implement the "re-routing" 😅 
We should probably add a test file, just ensuring all the deprecated methods still work (i.e. calling them all)